hangman/README.md

```diff
@@ -1,3 +1,7 @@
 8k students only:
 
 Describe how to run the multinode app.
+
+7345 Students, restart strategy:
+
+I would choose :permanent as the restart strategy. I would like to use GenSever to store the state. Because we need the server to be running until the game is over or the user exit the game.
```

Comment:
# -2

I normally deduct 3 for the answer :permanent. That's because the state is held in the server, and if the server crashes you've lost that state, and the game can't continue. However, I _think_ you're saying that you would also implement a separate server to store the state across restarts, hence the -2.

However, I still think :transient is correct here. If the game server exits normally, there's no point in restarting it, because there'd be no one connected to it.